.env.iv

```diff
@@ -1 +1 @@
-
�*�5e� ����V;
\ No newline at end of file
+�h���0�͘z�oH�
```

Comment:
This is wrong

Reply:
Ah you're right that square thing is supposed to be a question mark